prov/efa/src/rxr/rxr.h

```diff
@@ -83,9 +83,9 @@ static inline void rxr_poison_mem_region(uint32_t *ptr, size_t size)
 #endif
 
 /*
- * Set alignment to x86 cache line size.
+ * Set alignment to page size so that we can use MADV_DONTFORK.
  */
-#define RXR_BUF_POOL_ALIGNMENT	(64)
+#define RXR_BUF_POOL_ALIGNMENT	(ofi_get_page_size())
```

Comment:
Maybe drop the define and make the call directly where needed.  Is this change desired for v1.15.0rc1?

Reply:
No, we are still evaluating this change.